### DISCUSSION TITLE
Please bring back wire:transition

### DISCUSSION THREAD

**Discussion by GabrielSieben** [livewire/livewire#1876] [3. Feature ideas]
The `wire:transition` property, I feel, was a significant loss from v1.0. It was very simple - not as powerful as Alpine, and yet it could cover so many everyday use cases with much less code than Alpine. It worked great with any UI framework, Bootstrap or Tailwind.

Before, if I wanted to make a form animate in: `wire:transition="slide-in"`, and that's it. Maybe I could tweak it a bit, but I'm not a front-end designer. My client is mostly interested in having something that looks good _enough_, but having animation is icing on the cake for them. It's not worth my time to wrangle in Alpine transitions and my client isn't willing to pay for so much extra time to put Alpine everywhere, but if it was only one-liners like this, I'd use transitions far more often and I think my client would really appreciate it.

In my perfect world, `wire:transition` for fast, simple animations; and AlpineJS for fancier or more complex stuff.

**Comment by oliverbj** [comment]
Totally agree, @gjsman! I'm missing the simplicity of `wire:transition` in my current project also.

**Reply by DanielSpravtsev** [reply]
Agree, it was best solution out of the box

**Comment by kondratovbr** [comment]
Call me a dummy, but I still don't understand how exactly would I implement transitions for Livewire updates with Alpine.

I mean: Livewire updates something on the backend, receives the new HTML (let's say, some dialog window), now it will replace the old element with the new one in the DOM. We need to transition out the old element and transition in the new one. But how?
